public/app/core/components/RolePicker/RolePickerMenu.tsx

```diff
@@ -131,10 +136,10 @@ export const RolePickerMenu = ({
     }
   };
 
-  const onOpenSubMenu = (value: string) => {
+  const onOpenSubMenu = (groupType: 'fixed' | 'custom', value: string) => {
```

Comment:
```suggestion
  const onOpenSubMenu = (groupType: GroupType value: string) => {
```
There seems to be a type defined for this param.

Reply:
Oh yeah I missed to replace that one when adding the enum